modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBarLocale.vue

```diff
@@ -41,11 +86,30 @@ export default {
         modifiers: [ 'icon-right', 'no-motion' ],
         type: 'quiet'
       };
+    },
+    filteredLocales(input) {
+      return this.locales.filter(locale => {
+        return (
+          locale.label.toLowerCase().indexOf(this.search.toLowerCase()) > -1
+        );
+      });
     }
   },
   methods: {
+    isActive(locale) {
+      return window.apos.i18n.locale === locale.name;
+    },
+    isLocalized(locale) {
+      // TODO: Not sure we have the proper data to make this work properly yet.
```

Comment:
We do. Please see this PR from last week:

https://github.com/apostrophecms/apostrophe/pull/3274

I reckon it shouldn't be tough to integrate that with what you're doing.